File: csharp/sum-of-multiples/SumOfMultiples.cs
using System;
using System.Collections.Generic;
using System.Linq;

public static class SumOfMultiples
{
    public static int Sum(IEnumerable<int> multiples, int max)
        => Enumerable.Range(0, max).Where(c => multiples.Any(m => c % m == 0)).Sum();
}

File: ruby/sum-of-multiples/sum_of_multiples.rb
class SumOfMultiples
  def initialize(*nums)
    @mults = nums
  end

  def to(max)
    (0...max).select{|c| @mults.any?{|m| c % m == 0}}.sum
  end
end

module BookKeeping
  VERSION = 2
end
